view/src/components/foodbanks.js

```diff
@@ -0,0 +1,937 @@
+              {this.state.foodbankName}
+            </DialogTitle>
+            <DialogContent dividers>
+              <Chip
+                className={classes.chip}
+                label="High Food Insecurity"
+                size="small"
+              />
+              <Chip className={classes.chip} label="Major City" size="small" />
+              <Box
+                display="flex"
+                flexDirection="row"
+                flexWrap="wrap"
+                p={0}
+                m={0}
+              >
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Details:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Location of Food Bank:
+                    {`${this.state.location.substring(0, 30)}`}
+                    <br />
+                    {`${this.state.location.substring(30, 78)}`}
+                    Regrigeration Space (in pallets):
+                    {this.state.refrigerationSpaceAvailable}
+                    <br />
+                    Max Load Size (in pallets): {this.state.maxLoadSize}
+                  </Typography>
+                </Box>
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Point of Contact:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Name: {this.state.contactName}
+                    <br />
+                    Phone: {this.state.contactPhone}
+                    <br />
+                    Email: {this.state.contactEmail}
+                  </Typography>
+                </Box>
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Logistics:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Forklift: {this.state.forklift ? "yes" : "no"}
+                    <br />
+                    Pallet: {this.state.pallet ? "yes" : "no"}
+                    <br />
+                    Loading Dock: {this.state.loadingDock ? "yes" : "no"}
+                  </Typography>
+                </Box>
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Hours of Operation:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Monday-Friday: 9am - 5pm
+                    <br />
+                    Saturday: 10am - 4pm
+                    <br />
+                    Sunday: closed
+                  </Typography>
+                </Box>
+              </Box>
+            </DialogContent>
+          </Dialog>
+        </main>
+      );
+    }
+  }
+}
+
+const tagExamples = [
```

Comment:
Since this is deeply constant, it can be spelled `TAG_EXAMPLES`.  I usually put stuff like this at the top of the file rather than at the bottom, so someone reading the file will see its definition before they encounter its usage in code.

Reply:
Ideally this is a place holder until we load in all of the available tags, but for now I will make sure to have it formatted properly!